db/migrate/1599920640_remove_not_null_constraint_for_parity.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+    alter_table(:timetable_slots) do
+      set_column_allow_null :parity
```

Comment:
FYI: KOSapi never returns null in timetable slot’s (TTS) `period`. Many TTSs in KOS has period (`SUDYLICHY`) null when the period is actually “both”. For this reason, KOSapi transforms null period into `BOTH`. I’m thinking about changing it to return `UNDEFINED` (according to the [original design](https://kosapi.fit.cvut.cz/projects/kosapi/wiki/Parity])) when `weeks` is not null.

However, I agree with setting `parity` to null in Sirius instead of extending the `parity` enum type.